### ISSUE TITLE
product-slider element: wrong application of navigation settings

### ISSUE THREAD

**Issue by janobi** [shopware/platform#7810]
### PHP Version

8.3.19

### Shopware Version

6.6.10.2

### Affected area / extension

Platform(Default)

### Actual behaviour

In NEXT-39635, the product-slider element was reworked to get a new navigationArrows config instead of the old "navigation" one.

When the arrow-navigation is configured for a product-slider, the storefront template adds the same configured value to the sliderConfig for both the navigation arrows (controls) and the navigation dots (nav):
```
{% set productSliderOptionsSlider  = {
    controls: sliderConfig.navigationArrows.value in visibleValues,
    nav: sliderConfig.navigationArrows.value in visibleValues,
    ...
```

This leads to redundant navigation dots rendered in addition to the configured navigation arrows - in most cases, it will be occluded by the product-boxes, but you can find it in the DOM as:
```<div class="tns-nav" aria-label="Carousel Pagination">```

Additionally, the old "navigation" setting is still used in the template for the `has-nav` class when it should react to the new "navigationArrows" setting:
```{% if sliderConfig.navigation.value %} has-nav{% endif %}```

### Expected behaviour

When you set the navigationArrows config, that setting should be applied to the navigation arrows consistently, not also to the navigation dots.
I.e., `productSliderOptionsSlider` should be set to `nav: false,` and `sliderConfig.navigationArrows` instead of `sliderConfig.navigation` should be used for the `has-nav` class.

### How to reproduce

Add a product-slider with navigationArrows to a page, observe the output elements.

**Comment by marcelbrode** [comment]
Due to Accessibility and Styling Issues, this was already deactivated some time. We'll look into in the future again.

I'll use this ticket to fix the `{% if sliderConfig.navigation.value %} has-nav{% endif %}` issue instead.

Thanks for the contribution!
